package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "design-system",
-  "version": "0.14.4",
+  "version": "0.14.5”,
```

Comment:
After you fix it please check the VRT results to make sure there is no visual difference.